Dfe.PrepareTransfers.Web.Integration.Tests/BasicTests.cs

```diff
@@ -12,21 +13,22 @@ public class BasicTests : BaseIntegrationTests
         public BasicTests(IntegrationTestingWebApplicationFactory factory) : base(factory) { }
 
         [Theory]
-        [InlineData("/")]
+        [InlineData("/home")]
         public async Task Should_be_success_result_on_get(string url)
         {
             var projects = GetProjects();
 
             await OpenUrlAsync(url);
```

Comment:
Could you refactor this to use the following
private static string BuildRequestAddress(string path)
   {
      return $"https://localhost/{(path.StartsWith('/') ? path : $"/{path}")}";
   }

   protected async Task OpenAndConfirmPathAsync(string path, string expectedPath = null, string because = null)
   {
      await Context.OpenAsync(BuildRequestAddress(path));

      Document.Url.Should().Be(BuildRequestAddress(expectedPath ?? path), because ?? "navigation should be successful");
   }
  
  
  This will do an assertion on the URL after opening and would be good to have in the transfer codebase.